lib/resources/r4/related_person_patch.yaml

```diff
@@ -89,9 +98,39 @@ operations:
       <ul>
         <li>The <code>value.coding</code> must contain exactly one proprietary system.</li>
         <li>The <code>value.coding</code> cannot contain a standard system.</li>
+        <li>When specifying a `period`, the fields must include a time component with a timezone.</li>
         <li>Duplicate relationships may be added, but will only be represented once on the Resource.</li>
       </ul>
 
+  - name: test-relationship-id
+    path: /relationship/{index}/id
+    operation: test
+    type: string
+    url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
+    description: Test the `Relationship.id` value of the relationship at the given <code>{index}</code> in the list of related person relationships.
+    example: |
+      {
+        "op": "test",
+        "path": "/relationship/0/id",
+        "value": "CI-98278577-0"
+      }
+
+  - name: remove-relationship
```

Comment:
Did we lump the changes from OPENSVC-19819 in this PR?

Reply:
yeah this PR is a combination of two Jira's 18996&18873

Reply:
good to know. Thank you